knimin/lib/data_access.py

```diff
@@ -504,6 +507,8 @@ def format_survey_data(self, md):  # noqa
         -------
         dict of dict of dict
```

Comment:
Not necessary for this PR (since this was here before)

But we really need to specify what exactly the keys/values are.